code/game/gamemodes/demon/true_demon/_true_demon.dm

```diff
@@ -90,13 +90,11 @@
 /mob/living/carbon/true_devil/IsAdvancedToolUser()
 	return 1
 
-
 /mob/living/carbon/true_devil/canUseTopic()
 	if(stat)
 		return
 	return 1
 
-
```

Comment:
line spacings between procs are for readability.  The segmentation helps readers determen where procs are.

IIRC this has no effect on the compiled code so please don't remove it